%-------------------------------------------------------------------------------
%	SECTION TITLE
%-------------------------------------------------------------------------------
\cvsection{Research Experience}


%-------------------------------------------------------------------------------
%	CONTENT
%-------------------------------------------------------------------------------
\begin{cventries}
%---------------------------------------------------------


%---------------------------------------------------------

    \cventry
    {apr 2024 - present}
    {Research with Prof. Marinka Zitnik}
    {Harvard Medical School, Harvard University}{Cambridge, MA}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
       \item Building a causal foundation model synthesizing 280 million patient medical records and medical knowledge graphs integrating information across scales (molecules, proteins, genetics, pathways, and disease information) to predict patient counterfactuals.
       \item Developing new methods for conditional dynamic treatment effect estimation from a foundation model, with applications to 1) optimizing treatment policies and 2) generating clinically informed drug representations.
     \end{itemize}}\vspace{-4mm}

    \cventry
    {may 2024 - present}
    {Research with Profs. Jonathan Gootenberg and Omar Abudayyeh}
    {Harvard Medical School, Harvard University}{Cambridge, MA}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
    \item Developed novel approach for approximate optimal transport and flow matching for predicting effects of genetic perturbations, enabling generalization across unseen cell types.
    \item Designed and implemented a transformer-based models to develop embeddings for sets of cells to generate fine-grained cell-type embeddings, enabling better generalization across cell types.
    \item Led extensive benchmarking and validation efforts, systematically evaluating and unifying existing models for out-of-distribution prediction in single-cell sequencing using novel performance metrics.
         \end{itemize}}\vspace{-4mm}
     
    \cventry
    {june 2020 - sept 2024}
    {Research with Prof. Kosuke Imai}
    {Institute for Quantitative Social Science, Harvard University}{Cambridge, MA}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
       \item Constructed a non-parametric framework for causal inference under general interference using regression.
       \item Developed efficient algorithms and theory for non-parametric monotone regression for estimation and inference in the general interference framework, with applications in large-scale network and spatial experiments.
       \item Worked on adaptive methods for pure exploration in combinatorial designs in generalized linear models.
     \end{itemize}}\vspace{-4mm}

    \cventry
    {sept 2021 - june 2023}
    {Research in OxCSML Group under Yee Whye Teh}
    {Statistics Department, University of Oxford}{Oxford, England}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
        \item Built novel models of cyclic peptides and antibody loops using constrained diffusion and flow-matching models.
        \item Developed techniques for extending diffusion models to constrained domains in general Riemannian geometries.
     \end{itemize}}\vspace{-4mm}

    \cventry
    {june 2021 - sept 2023}
    {Research Fellow}
    {Center for Science of Science \& Innovation, Northwestern University}{Evanston, IL}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
        \item Synthesized huge international datasets on scientific grantmaking and publication to generate a picture of the landscape and political implications of scientific funding.
         \item Leveraged LLMs to develop fine-grained topic classifications across grants and paper abstracts to understand when grant-making institutions lead scientific investigation as opposed to following what is already being studied.
     \end{itemize}}\vspace{-4mm}

    \cventry
    {june 2021 - june 2022}
    {Research Assistant for Prof. Victor Chernozhukov}
    {Economics Department, Massachusetts Institute of Technology}{Cambridge, MA}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
        \item  Developed methods for machine learning dynamic treatment effects for improved policy learning in healthcare. 
        \item Specifically extended g-estimation to non-linear blip functions using tools from minimax optimization.
    \end{itemize}}\vspace{-4mm}    
     
 %\cventry
 %   {spring 2021}
 %   {Research Assistant for Prof. Aliya Saperstein and Prof. Michelle Jackson}
 %   {Sociology Department, Stanford University}{Stanford, CA}
 %   {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
 %         \item Constructed a timeline of major developments, conflicts, and characters in the history of statistics and eugenics.
 %         \item Developed the hypothesis that from 1850-1930 economics, psychology, and sociology embraced biological determinism, quantification, and professionalization to secure legitimacy and funding.
 %         \item Documented how statistical controversies drove eugenics and, perhaps more surprising, how eugenic controversies drove statistical innovation, through in-depth engagement with primary sources.
 %    \end{itemize}}\vspace{-4mm}

 \cventry
    {jan 2020 - june 2021}
    {Research Intern for Prof. Jure Leskovec and Prof. David Grusky}
    {Stanford Center on Poverty and Inequality and Stanford Network Analysis Project}{Stanford, CA}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
       \item Used path crossings to study socioeconomic stratification in the United States, developing insights about heterogeneity in segregation across different social contexts and across different income deciles.
      \item Worked on algorithms to identify when individuals cross paths in time and space using massive GPS data.
     \end{itemize}}\vspace{-4mm}
    
 \cventry
    {jan 2018 - june 2021}
    {Research Intern for Prof. Anshul Kundaje}
    {Kundaje Lab, Stanford AI Lab}{Stanford, CA}
    {\vspace{-4mm}
     \begin{itemize}[leftmargin=5mm]
          \item Created modular system for generating DNA sequences using GANs/VAEs/Transformers and optimizing these models to produce sequences with specified properties (level of gene expression or chromatin accessibility).
          \item Developed a k-NN algorithm to evaluate (1) the fidelity of samples from generative models and (2) the robustness of neural network predictions on regression outputs (extending existing work on classification).
          %\item Constructed neural net for predicting protein expression from whole genome ChIP-exo data. Used causal inference methods to extract and build a synthetic DNA regulatory logic for simulations in generative genomics.
          %\item Contributed to SimDNA, a Python library for simulating DNA datasets to evaluate machine learning methods.
          %\item Built a novel method for generating regulatory DNA to achieve targeted levels of protein expression through augmenting conditional GAN architectures (CS229 best project).
     \end{itemize}}\vspace{-4mm}
     
    %\cventry
    %{fall 2020}
    %{Research Assistant for Prof. Josh Kleinfeld}
    %{Pritzker School of Law, Northwestern University}{Chicago, IL}
    %{\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
    %      \item Used network models to explain plea bargaining's diffusion and its impacts on global criminal justice systems.
    %      \item Conducted research on various questions of legal/intellectual history.
    %\end{itemize}}\vspace{-4mm}
     
    
    %\cventry
    %{apr - dec 2019}
    %{Research assistant for Prof. Mark Grief}
    %{English Department, Stanford University}{Stanford, CA}
    %{\vspace{-4mm}\begin{itemize}[leftmargin=5mm] 
    %\item Conducted literature reviews on topics including social theory, psychology, and gender and sexuality studies. 
    %\end{itemize}}\vspace{-4mm}

  %cventry
  %  {dec 2015 - sept 2017}
  %  {Research Fellow}
  %  {Ostrander Lab, National Human Genome Research Institute}{Bethesda, MD}
  %  {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
%	      \item Responsible for building a DNA database to facilitate access and analysis of structural variants.
%	      \item Used random forest approach to identify candidate cancer risk genes from SNP arrays.
   %  \end{itemize}}\vspace{-4mm}

%---------------------------------------------------------
\end{cventries}


\cvsection{Work Experience}


%-------------------------------------------------------------------------------
%	CONTENT
%-------------------------------------------------------------------------------
\begin{cventries}
%---------------------------------------------------------


%---------------------------------------------------------

    \cventry
    {summer 2022}
    {Research Intern}
    {Knowledge Graph Team, Google}{San Fransisco, CA}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
        \item Research integrating LLMs and knowledge graphs (KGs) for search, particularly combining LLM representations with graph neural networks over a knowledge graph to produce KG-informed representations.
        \item Developed methods to identify author expertise using LLM/KG representations for multimedia topic modeling.
     \end{itemize}}\vspace{-4mm}

  \cventry
    {june 2018 - may 2021}
    {Data Scientist}
    {Data for Progress}{New York, NY}
    {\vspace{-4mm}
     \begin{itemize}[leftmargin=5mm]
     	  \item Led development of polling infrastructure that produced the most accurate poll results in the Democratic Primary. Built out MySQL database for storage and analysis of survey responses. Automated chart and report generation from this database. Developed search engine and website for internal use to assist in research.
     	  \item Designed, conducted, and analyzed polls used to guide policy change for the Green New Deal, Medicare for All, several HR-1 issues, and criminal justice reform, among other progressive issues.
          \item Developed novel poll weighting scheme achieving state-of-the-art accuracy.
          \item Automated argument detection so that non-technical colleagues to easily interpret open-ended survey responses in policy briefs by building hierarchical Dirichlet process models for non-parametric topic modeling.
          \item Created an ecological loss function and corresponding neural networks, extending ecological inference.
        \item Led team creating word2vec models to analyze gender/racial bias in news articles around the 2016 election.
     \end{itemize}}\vspace{-4mm}
     
    \cventry
    {fall 2020}
    {Data Science Consultant}
    {Sunrise Movement}{Washington, DC}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
          \item Used LASSO and random forest algorithms to build interpretable heuristics for identifying low-propensity swing state voters for a large-scale get-out-to-vote campaign in advance of the 2020 presidential election. 
     \end{itemize}}\vspace{-4mm}
     
    %\cventry
    %{fall 2020}
    %{Research Assistant for Prof. Josh Kleinfeld}
    %{Pritzker School of Law, Northwestern University}{Chicago, IL}
    %{\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
    %      \item Used network models to explain plea bargaining's diffusion and its impacts on global criminal justice systems.
    %      \item Conducted research on various questions of legal/intellectual history.
    %\end{itemize}}\vspace{-4mm}
     
    \cventry
    {summer 2018}
    {Machine Learning Specialist}
    {Star Lab Corporation}{Washington, DC}
    {\vspace{-4mm}\begin{itemize}[leftmargin=5mm]
          \item Worked on a Red Hat kernel module to log system activity and leveraged it for neural net anomaly detection.
     \end{itemize}}\vspace{-4mm}

%---------------------------------------------------------
\end{cventries}
